test/functional/sqlsrv/MsSetup.inc

```diff
@@ -44,4 +44,11 @@ if (isset($_ENV['MSSQL_SERVER']) || isset($_ENV['MSSQL_USER']) || isset($_ENV['M
 $keystore = "none";         // key store provider, acceptable values are none, win, ksp, akv
 $dataEncrypted = false;     // whether data is to be encrypted
 
+// for Azure Key Vault
+$AKVKeyStoreAuthentication = 'TARGET_AKV_AUTH';   // can be KeyVaultPassword or KeyVaultClientSecret
+$AKVPrincipalName = 'TARGET_AKV_PRINCIPAL_NAME';  // for use with KeyVaultPassword
```

Comment:
Thus, based on $keystore entry in MsSetup.inc, the value of $AKVKeyStoreAuthentication will be updated in Test Lab